axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveCreateFromInvoiceServiceImpl.java

```diff
@@ -144,9 +144,12 @@ public Move createMove(Invoice invoice) throws AxelorException {
             company.getName());
       }
 
-      log.debug(
-          "Creation of a move specific to the invoice {} (Company : {}, Journal : {})",
-          new Object[] {invoice.getInvoiceId(), company.getName(), journal.getCode()});
+      if (journal != null) {
+        log.debug(
+                "Creation of a move specific to the invoice {} (Company : {}, Journal : {})",
+                new Object[] {invoice.getInvoiceId(), company.getName(), journal.getCode()});
+      }
+
```

Comment:
Here it's just a debug message, so I guess it's fine to just use an Optional like : 
Optional.ofNullable(journal).map(Journal::getCode).orElse("")
Instead of not displaying the whole line